src/components/datacollection/DataPageMenu.svelte

```diff
@@ -0,0 +1,78 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<style>
+  button:disabled {
+    cursor: default;
+    color: grey;
+  }
+</style>
+
+<script lang="ts">
+  import { t } from '../../i18n';
+  import { createMenu } from 'svelte-headlessui';
+  import ClearIcon from 'virtual:icons/ri/delete-bin-2-line';
+  import UploadIcon from 'virtual:icons/ri/upload-2-line';
+  import DownloadIcon from 'virtual:icons/ri/download-2-line';
+  import ArrowDownIcon from 'virtual:icons/ri/arrow-down-s-line';
+  import MenuItems from '../control-bar/control-bar-items/MenuItems.svelte';
+  import MenuItem from '../control-bar/control-bar-items/MenuItem.svelte';
+  import MenuTransition from '../MenuTransition.svelte';
+
+  export let downloadDisabled = false;
+  export let clearDisabled = false;
+  export let onClearGestures: () => void;
+  export let onDownloadGestures: () => void;
+  export let onUploadGestures: () => void;
+
+  const menu = createMenu({ label: $t('dataMenu.label') });
+
+  const onSelect = (event: Event) => {
+    const { selected } = (event as CustomEvent).detail;
+    switch (selected) {
+      case 'upload-gestures': {
```

Comment:
This should probably have a common suffix or no suffix at all. The props are all fooGestures.